app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt

```diff
@@ -692,7 +705,7 @@ fun CommentFooterLine(
     var showMoreOptions by remember { mutableStateOf(false) }
 
     val canMod =
-        remember {
+        remember(admins) {
```

Comment:
Need your help with this one. 

This was an error with the last PR. On first app load, it would incorrectly not should the extra mod actions, because admins hadn't loaded yet. 

This fixes it, but it might cause a lot of recompositions when admins show up? I'm having trouble deciphering when to use derivedState vs remember(key). 

Also, should this remember also include account, IE `remember(account, admins) { ...`

Reply:
>  I'm having trouble deciphering when to use derivedState vs remember(key).

derivedState is to prevent recompositions when the result is the same. Here i have rather that the computation (the function call itself) is skipped as much as possible too. So I prefer a remember.

> This fixes it, but it might cause a lot of recompositions when admins show up

As long as the key doesn't change it shouldn't cause any recompositions, so as long as the list stays the same it shouldnt be a problem.

> Also, should this remember also include account

if this needs to change when the account changes then the account needs to be included in the key. But I think account change would force it to fully rebuild the compose tree due to navigation. So it wouldn't be necessary

Reply:
Gotcha, thx.